src/Form/Formbot.example.js

```diff
@@ -68,7 +68,7 @@ export default function() {
       }}>
       {({ values, onSubmit, onChange, errors, onBlur }) => (
         <form onSubmit={onSubmit}>
-          <FormGroup>
+          <FormGroup legend="A Group of Inputs">
```

Comment:
If `values.name` is `undefined`, I think the input still expects the `value` to be of type `string`, so we will likely want to set the value to empty string if it is undefined? Othewise a proptype error results.

Reply:
It looks like the message field doesn't specify the value prop, is that an option here?